app/code/community/Ebizmarts/MailChimp/etc/system.xml

```diff
@@ -37,7 +37,8 @@
                             <label>Enabled</label>
                             <frontend_type>select</frontend_type>
                             <source_model>adminhtml/system_config_source_yesno</source_model>
-                            <comment><![CDATA[By enabling this extension, you agree to <a href="https://mailchimp.com/legal/additional-terms/" target="_blank">Mailchimp's additional terms</a>.]]></comment>
+                            <comment>
```

Comment:
This blank lines after the start tag cause failure.

Reply:
Solved.